.ci/build_wheel.py

```diff
@@ -0,0 +1,94 @@
+# This script generates the different versions of the ansys-dpf-core wheels based on a given input.
+# Input can be one of ["any", "win", "manylinux1", "manylinux_2_17"]
+
+import argparse
```

Comment:
if I remember correctly, wheels used to be build by dpf-action repo, what is done where now?

Reply:
what about pydpf-post?

Reply:
> if I remember correctly, wheels used to be build by dpf-action repo, what is done where now?

@cbellot000 they indeed used to be created by an action in the dpf-action repo. Now I've changed to have the logic directly in the pydpf-core yml files. I will consider making a common action later but since this is particular to pydpf-core, it would not have a big advantage to move it to the pydpf-actions

Reply:
> what about pydpf-post?

Right now the pydpf-post regular CI just clones the pydpf-core repo and installs it as-is since the dependency is on the git repository directly. I'll have to make a PR on PyDPF-Post to properly build the ansys-dpf-core wheel based on the context, but it can be done after this.
The pydpf-post Release CI gets an ansys-dpf-core wheel from pip, so it requires no change.

Reply:
Okay, will there be any time with a broken pipeline in pydpf-post? (it's just to make sure we capture evrything)

Also if this will be used in pydpf-post, I guess you'll move it to pydpf-actions after?

Reply:
> Okay, will there be any time with a broken pipeline in pydpf-post? (it's just to make sure we capture evrything)
> 
> Also if this will be used in pydpf-post, I guess you'll move it to pydpf-actions after?

Hi @cbellot000, sorry I do not understand the first sentence.

As for pydpf-post, indeed once I have a build_core action I can move it to the pydpf-actions, but this is really a consolidation step which is not really required at this point.

Reply:
See [here](https://github.com/ansys/pydpf-post/actions/runs/6479253578) for a run of the PyDPF-Post CI while targeting this branch specifically in the requirements, meaning this branch will be cloned when pip installing dependencies of `ansys-dpf-post`.